src/main/java/vat/VatTaxRatio.java

```diff
@@ -0,0 +1,47 @@
+package vat;
+
+public enum VatTaxRatio {
+    AT_STANDARD(Country.AUSTRIA, VatKind.STANDARD, 1.20d),
```

Comment:
A good idea, but instead of having x4 combos for every vat kind, how about having a combo per country and passing the exact kind values inside instead?
```
public enum VatTaxRatio
{
  Lithuana(Country.Lithuania, vat1, vat2, vat3, vat4)
}
```